xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -732,7 +738,15 @@ class CVideoPlayActionProcessor : public CVideoPlayActionProcessorBase
     const ContentUtils::PlayMode mode{m_item.GetProperty("CheckAutoPlayNextItem").asBoolean()
                                           ? ContentUtils::PlayMode::CHECK_AUTO_PLAY_NEXT_ITEM
                                           : ContentUtils::PlayMode::PLAY_ONLY_THIS};
-    VIDEO_UTILS::PlayItem(std::make_shared<CFileItem>(m_item), "", mode);
+
+    if (m_item.GetVideoInfoTag()->m_type == MediaTypeMovie)
```

Comment:
Could you please add a `//! @todo get rid of special handling for movie versions`, so we can pick this up in a follow-up PR?